fetchit_challenge/models/rarelab_long_table/model.sdf

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" ?>
+<sdf version='1.4'>
+  <model name="rarelab_long_table">
+      <static>1</static>
+      <link name='link'>
+        <inertial>
+          <mass>4.1</mass>
+          <inertia>
+              <ixx>0.304192</ixx>
+              <ixy>0.0</ixy>
+              <ixz>0.0</ixz>
+              <iyy>0.966527</iyy>
+              <iyz>0.0</iyz>
+              <izz>0.916854</izz>
+            </inertia>
+        </inertial>
+        <collision name='collision'>
+          <geometry>
+            <box>
+              <size>1.524 0.61 0.75</size>
```

Comment:
What are those values? Should this be used: https://github.com/TheRARELab/fetch_gazebo/blob/gazebo9/fetchit_challenge/models/fetchit_table/model.sdf#L17-L32?